src/segments/executiontime_test.go

```diff
@@ -289,3 +290,45 @@ func TestExecutionTimeFormatDurationRound(t *testing.T) {
 		assert.Equal(t, tc.Expected, output)
 	}
 }
+
+func TestExecutionTimeFormatDurationLucky7(t *testing.T) {
+	cases := []struct {
+		Input    string
+		Expected string
+	}{
+		{Input: "0.001s" /*    */, Expected: "    1ms"},
```

Comment:
if you add these properties to their own line, it solves the alignment too.

```suggestion
		{
		    Input:        "0.001s",
		    Expected: "    1ms"
		},
```

Reply:
I'd rather keep it with the inline comments. Do you want it in new lines?

Reply:
the reason I'd add newlines myself is because will format correctly aligning everything out of the box while this requires someone adjusting alignments.

So I would add the newlines.